### PR TITLE
Using removable Pod instead of Deployment in HPA example

### PR DESCRIPTION
**Pull request by butuzov**
Enhancement for the page [Tasks / Run Applications / Horizontal Pod Autoscaler Walkthrough](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#increase-load)

Using removable Pod instead of Deployment in HPA example.

### CODE REVIEW

**Comment by butuzov** [comment]
change is here -> https://deploy-preview-17604--kubernetes-io-master-staging.netlify.com/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#increase-load

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Now, we will see how the autoscaler reacts to increased load.
 We will start a container, and send an infinite loop of queries to the php-apache service (please run it in a different terminal):
 
 ```shell
-kubectl run -i --tty load-generator --image=busybox /bin/sh
+kubectl run -it --rm --tty load-generator --image=busybox --restart=Never /bin/sh
```
Comment (sftim) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md]:
```suggestion
kubectl run --generator=run-pod/v1 -i --rm --tty load-generator --image=busybox --restart=Never /bin/sh
```
- `-t` and `--tty` achieve the same thing
- specifying `run-pod/v1` generator for clarity.

Comment (butuzov) [reply content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md]:
```suggestion
k run --generator=run-pod/v1 -it --rm load-generator --image=busybox /bin/sh
```
Miss it. Here even shorter variance.

**Pull request merged**